crates/wash-cli/src/build.rs

```diff
@@ -68,15 +72,29 @@ pub async fn handle_command(command: BuildCommand) -> Result<CommandOutput> {
                 })
             };
 
-            let actor_path = build_project(&config, sign_config)?;
+            let actor_path = if command.sign_only {
+                std::env::set_current_dir(&config.common.path)?;
+                let signed_path = sign_actor_wasm(
+                    &config.common,
+                    actor_config,
+                    sign_config.context("cannot supply --build-only and --sign-only")?,
```

Comment:
This definitely shouldn't be possible with the conflicts. I just did it because the unwrap here feels wrong, e.g. may as well save face here with a nice error, but I can just leave a SAFETY comment if that's preferred 😄